### PR TITLE
[ISSUE #3928] Optimize http protocol log print

### LINKED ISSUE #3928: [Enhancement] Optimize http protocol log print
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/eventmesh/issues?q=is%3Aissue) and found no similar issues.

### Enhancement Request

image
A large number of invalid log prints lead to a large number of string allocations affecting performance.

### Describe the solution you'd like

Optimize http protocol log print

### Are you willing to submit PR?

- [X] Yes I am willing to submit a PR!

### PR DESCRIPTION
**Pull request by mxsm**
Fixes #3928 

### Motivation

*Explain the content here.*
*Explain why you want to make the changes and what problem you're trying to solve.*

### Modifications

*Optimize http protocol log print*

### Documentation

- Does this pull request introduce a new feature? (yes / no)
- If yes, how is the feature documented? (not applicable / docs / JavaDocs / not documented)
- If a feature is not applicable for documentation, explain why?
- If a feature is not documented yet in this PR, please create a followup issue for adding the documentation

### CODE REVIEW

**Comment by xwm1992** [comment]
@mxsm please fix the conflict.

**Comment by mxsm** [comment]
@xwm1992 PTAL~ I have fixed it

**Comment by mxsm** [comment]
@xwm1992 PTAL~ I have fixed it

File: eventmesh-runtime/src/main/java/org/apache/eventmesh/runtime/util/TraceUtils.java [hunk 1]
```diff
@@ -106,18 +100,21 @@ public static void finishSpan(ChannelHandlerContext ctx, CloudEvent event) {
...
     }
 
-    public static void finishSpanWithException(Span span, Map<String, Object> map, String errMsg,
-        Throwable e) {
+    public static void finishSpanWithException(Span span, Map<String, Object> map, String errMsg, Throwable e) {
         try {
-            log.debug("finishSpanWithException with map:{}", map);
+            if (log.isWarnEnabled()) {
+                log.warn("FinishSpanWithException with map:{}", map);
+            }
```
Comment (Pil0tXia) [inline eventmesh-runtime/src/main/java/org/apache/eventmesh/runtime/util/TraceUtils.java:117]: May I ask that why logging level should be upgraded to `warn` here?

By the way, `log.isWarnEnabled()` is redundant~

**Pull request closed without merging**